syntax = "proto3";
package msg;
message sendMessage
{
    int32 messageID = 1;
    bytes data = 2;
	bool isKey = 3;
}